vision/text/detect_words.py

```diff
@@ -41,34 +42,36 @@ def detect_russian_word(
         A list of bounding box objects that contain desired text
             Note that there may be different bounding boxes for each word
         """
-        sliced_rotated_image, x_ul, y_ul = self._get_rotated_min_area_rect(color_image, depth_image)
+        sliced_rotated_image, x_ul, y_ul = self._get_rotated_min_area_rect(
+            color_image, depth_image
+        )
 
         if len(sliced_rotated_image) == 0:
             return []
 
-        tessdata = pytesseract.image_to_data(
+        self.tessdata = pytesseract.image_to_data(
```

Comment:
`self.tessdata` should be set to empty at the start of this function so that if the function returns at `if len(sliced_rotated_image) == 0:` the tessdata isn't from the previous image.